litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/DashboardCloneModal/index.tsx

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
same

Reply:
same here we cannot return as logic does not need that.